app/2.0/docs/devguide/dom-template.md

```diff
@@ -38,18 +55,125 @@ Example: { .caption }
+```js
+(function() {
+  let memoizedTemplate;
+
+  class MyExtension extends MySuperClass {
+    static get template() {
+      if (!memoizedTemplate) {
+        // retrieve superclass template
+        memoizedTemplate = MySuperClass.template;
+        // add a node to the template.
+        let div = document.createElement('div');
+        div.textContent = 'Hello from an extended template.'
+        memoizedTemplate.content.appendChild(div);
+      }
+      return memoizedTemplate;
+    }
+  }
+
+})();
+```
+
+The following example shows a more complicated could wrap a superclass template with a child
+template.
+
+```html
+<dom-module id="my-ext">
+    <template>
+      <h2>Extended template</h2>
+      <!-- superclass template will go here -->
+      <div id="footer">
+        Extended template footer.
+      </div>
+    </template>
+  <script>
+    (function() {
+      let memoizedTemplate;
+
+      class MyExtension extends MySuperClass {
+
+        static get is() { return 'my-ext'}
+        static get template() {
+          if (!memoizedTemplate) {
+            // Retrieve this element's dom-module template
+            memoizedTemplate = Polymer.DomModule.import(this.is, 'template');
+
+            // Clone the contents of the superclass template
+            let superTemplateContents = document.importNode(MySuperClass.template.content, true);
+
+            // Insert the superclass contents
+            let footer = memoizedTemplate.content.querySelector('#footer');
+            memoizedTemplate.content.insertBefore(superTemplateContents, footer);
+          }
+          return memoizedTemplate;
+        }
+      }
+      customElements.define(MyExtension.is, MyExtension);
+    })();
+  </script>
+</dom-module>
+```
+
+### Elements with no shadow DOM
+
+To create an element with no shadow DOM, don't specify a DOM template (either using a `<dom-module>`
+or by overriding the `template` getter), then no shadow root is created for the element.
+
+If the element is extending another element that has a DOM template and you don't want a DOM template,
+define a `template` getter that returns a falsy value.
+
 ### URLs in templates {#urls-in-templates}
 
-A URL in a template may be relative to an application or to a specific component. For example,
-if a component includes images alongside an HTML import that defines an element, the image URL needs
-to be resolved relative to the import. However, an application-specific element may need to include
-links to URLs relative to the main document.
+A relative URL in a template may be relative to an application or to a specific component. For
+example, if a component includes images alongside an HTML import that defines an element, the image
+URL needs to be resolved relative to the import. However, an application-specific element may need
+to include links to URLs relative to the main document.
 
 To ensure URLs resolve properly, Polymer provides two properties that can be used in data bindings:
```

Comment:
Maybe it would be good to say "By default, Polymer does not modify URL's in templates, so all relative URL's will be relative to the main document URL. This is because when the template content is cloned and added to the main document, the browser evaluates the URL's as relative to the document (not the original location of the template)."